ibis/expr/types/relations.py

```diff
@@ -2956,10 +2963,11 @@ def describe(
                 min=col_min,
                 **quantile_values,
                 max=col_max,
+                pos=lit(pos),
             )
             aggs.append(agg)
 
-        t = ibis.union(*aggs)
+        t = ibis.union(*aggs).order_by(ibis.asc("pos")).drop("pos")
```

Comment:
Ordering and then dropping is logically a no-op when any subsequent operations are performed. I think it's fine to include the column position in the output and remove this order -> drop call sequence.